apps/web/pages/signup.tsx

```diff
@@ -278,15 +310,21 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
     username = available ? username : suggestion || username;
   }
 
+  const isValidEmail = checkValidEmail(verificationToken.identifier);
+  const isOrgInviteByLink = orgSlug && !isValidEmail;
+
```

Comment:
when inviting user by copy link the identifier is something like "invite-link-for-teamId-2" which is not an email because we don't know the email yet